build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
`OUR` is indicating it's the one we built. Not the system one. I'm open to suggestions to another word, but I don't think we should remove the implication that this is the one we built. For the warmup experiment It's super important that we don't compile using any system compilers (other than for the initial gcc bootstrap).